packages/json-react-layouts-data-loader/src/index.tsx

```diff
@@ -175,6 +213,18 @@ export function init<Services extends object>(
             )
 
             if (dataDefinition) {
+                if (dataDefinition.useRuntimeParams) {
```

Comment:
If this is specified, it renders a different component, meaning the old (without useRuntimeParams) is unmounted so there is a new hooks stack